perses/app/relative_setup.py

```diff
@@ -811,14 +811,14 @@ def _solvate_system(self, topology, positions, model='tip3p',phase='complex', bo
         #modeller.delete(hs)
         #modeller.addHydrogens(forcefield=self._system_generator.forcefield)
         _logger.info(f'box_dimensions: {box_dimensions}')
-        _logger.info(f'solvent padding: {self._padding._value}')
+        _logger.info(f'solvent padding: {self._padding}')
```

Comment:
I have no idea how our tests never caught this, but `self._padding` doesn't ahe a `_value` attr, ~2 year old bug?